include/gsl/string_span

```diff
@@ -108,18 +108,20 @@ namespace details
 // Will fail-fast if sentinel cannot be found before max elements are examined.
 //
 template <typename T, const T Sentinel>
-span<T, dynamic_extent> ensure_sentinel(T* seq, std::ptrdiff_t max = PTRDIFF_MAX)
+span<T, dynamic_extent> ensure_sentinel(T* seq,
```

Comment:
Preexisting perf: probably special casing a 1 overload version instead of default arguments will win